jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextAreaExampleTests.java

```diff
@@ -0,0 +1,49 @@
+package io.github.epam.example.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.html5Page;
+import static io.github.com.pages.HtmlElementsPage.disabledTextArea;
+import static io.github.com.pages.HtmlElementsPage.textArea;
+import static io.github.epam.html.tests.site.steps.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+
+public class TextAreaExampleTests implements TestsInit {
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        html5Page.shouldBeOpened();
+        textArea.clear();
+    }
+
+    String text = "TextArea";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4698334)